files/galaxy/dynamic_job_rules/production/total_perspective_vortex/tools.yml

```diff
@@ -1375,3 +1375,13 @@ tools:
     cores: 5
   .*data_manager_cat.*:
     cores: 5
+    
+  # cactus suite
+  toolshed.g2.bx.psu.edu/repos/galaxy-australia/cactus_cactus/cactus_cactus/.*:
+    cores: 16
+    mem: 32
```

Comment:
you can leave out `mem: 32` and get the default 3.8xcores, unless this ratio is important

Reply:
thanks, that was just when we were trying to find somewhere for it to run on dev.